src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Models/RazorCodeAction.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using MediatR;
+using Newtonsoft.Json;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models
+{
+    [JsonObject]
+    public class RazorCodeAction : CodeAction, IRequest<RazorCodeAction>, IBaseRequest
```

Comment:
Inherits from `CodeAction` to ensure we're still able to put this into the O# `CommandOrCodeActionContainer`

Reply:
```suggestion
    internal class RazorCodeAction : CodeAction, IRequest<RazorCodeAction>, IBaseRequest
```